api

```diff
@@ -2858,6 +2858,8 @@ If this error keeps happening, try installing the More RAM app from Pi-Apps. Fin
   #return the information
   local IFS=$'\n'
   echo -e "${error_type}\n${error_caption[*]}"
+
+  rm -f "$tmplogfile"
```

Comment:
Where is this variable set? I don't see it being assigned anywhere.

Reply:
Sorry. Artifact of a previous refactor attempt.